python/rascaline/rascaline/utils/clebsch_gordan/_equivariant_power_spectrum.py

```diff
@@ -0,0 +1,201 @@
+"""
+This module provides a convenience calculator for computing a single-center equivariant
+power spectrum.
+"""
+
+from typing import List, Optional, Union
+
+from ...calculators import SphericalExpansion
+from ...systems import IntoSystem
+from .._backend import Device, DType, Labels, TensorMap, TorchModule, operations
+from ._cg_product import ClebschGordanProduct
+from ._density_correlations import _filter_redundant_keys
+
+
+class EquivariantPowerSpectrum(TorchModule):
```

Comment:
```suggestion
class EquivariantSoapPowerSpectrum(TorchModule):
```

Should we refer to SOAP here? it makes the class name a bit longer, but I could live with it

Reply:
I'm super convinced about this one, but happy for you to decide